utils/pvacapi/config/swagger.yaml

```diff
@@ -1035,6 +1035,50 @@ paths:
           description: "Unexpected error"
           schema:
             $ref: "#/definitions/Error"
+  /api/v1/export/{processID}:
+    get:
+      tags:
+        - "export"
+      summary: "export a process"
+      description: "Copies the results of a process to the export folder"
+      operationId: "utils.pvacapi.controllers.processes.export"
+      parameters:
+        - name: "processID"
+          in: "path"
+          description: "Id of the process to export"
+          required: true
+          type: "integer"
+      responses:
+        200:
+          description: "No meaningful data to return"
+          schema:
+            type: "string"
+        default:
+          description: "Unexpected error"
+          schema:
+            $ref: "#/definitions/Error"
+  /api/v1/delete/{processID}:
```

Comment:
Analogous to above, this should be `/api/v1/process/{processID}/archive` and should be a DELETE request instead of GET.